configure/RELEASE.local

```diff
@@ -30,8 +30,11 @@ TIMINGAPI_MODULE_VERSION = R0.6
 # FOO = /Full/Path/To/Development/Version 
 # ==========================================================)
 ASYN = $(EPICS_MODULES)/asyn/$(ASYN_MODULE_VERSION)
-EVRCLIENT = $(EPICS_MODULES)/evrClient/$(EVRCLIENT_MODULE_VERSION)
-TIMINGAPI = $(EPICS_MODULES)/timingApi/$(TIMINGAPI_MODULE_VERSION)
+
+ifeq ($(findstring pcds,$(FACILITY_ROOT)),)
```

Comment:
Any reason why this is just `pcds` and not `/reg/g/pcds`? Either way works though, I suppose.